css/index.css

```diff
@@ -111,6 +149,138 @@ p {
     align-items: center;
     justify-content: center;
 }
+ #box1{
+    background: teal;
+}
+ #box2{
+    background: gold;
+}
+ #box3{
+    background: cadetblue;
+}
+ #box4{
+    background: coral;
+}
+ #box5{
+    background: crimson;
+}
+ #box6{
+    background: forestgreen;
+}
+ #box7{
+    background: darkorchid;
+}
+ #box8{
+    background: hotpink;
+}
+ #box9{
+    background: indigo;
+}
+ #box10{
+    background: dodgerblue;
+}
+
+
```

Comment:
Great work recognizing the parent/child relationship and only overriding what needs to be changed in the child.